### PR TITLE
bump go-utils version

### PR DESCRIPTION
**Pull request by jehawley**
Bump go-utils version (for https://github.com/solo-io/build-bot/issues/73)

### CODE REVIEW

File: .github/workflows/pull_request.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    - uses: azure/setup-kubectl@v1
+      with:
+        version: 'v1.18.0'
+    - uses: engineerd/setup-kind@v0.3.0
+    - name: Build
+      run: |
+          go build ./...
+    - name: Run tests
+      env:
+        GITHUB_TOKEN: ${{ secrets.CLOUDBUILD_GITHUB_TOKEN }}
```
Comment (kdorosh) [inline .github/workflows/pull_request.yaml]: this secret doesn't exist in this repo, we can probably remove this

**Review by kdorosh — DISMISSED** [review]

File: .github/workflows/pull_request.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+      with:
+        version: 'v1.18.0'
+    - uses: engineerd/setup-kind@v0.3.0
+    - name: Build
+      run: |
+          go build ./...
+    - name: Run tests
+      run: |
+        go get -v github.com/onsi/ginkgo/ginkgo@v1.12.0 && export PATH=$PATH:$(go env GOPATH)/bin/
+        ginkgo  -r   -p   -failFast   -randomizeSuites   -randomizeAllSpecs   -skipPackage=./installutils/kubeinstall,./debugutils/test
```
Comment (kdorosh) [inline .github/workflows/pull_request.yaml]: we don't need the skip package flag anymore

Comment (jehawley) [reply .github/workflows/pull_request.yaml]: We do, actually - it doesn't pass when I remove it.

Comment (jehawley) [reply .github/workflows/pull_request.yaml]: Though I haven't tried it again since fully upgrading go-utils, so let's see what happens with this next run.

**Review by kdorosh — APPROVED** [review]

**Pull request merged**